src/components/common/data-not-found/data-not-found.directive.coffee

```diff
@@ -0,0 +1,63 @@
+module = angular.module('impac.components.common.data-not-found',[])
+
+module.directive('commonDataNotFound', ($templateCache, $log, $http, ImpacAssets, ImpacTheming, ImpacMainSvc) ->
+  return {
+    restrict: 'A',
+    scope: {
+      isKpi: '=?'
```

Comment:
instead of `isKpi`, I would recommend to propose a more general parameter like for example `hasImage` => this way you can use the directive for any component which can be a widget, a kpi, or anything else...